backend/.air.toml

```diff
@@ -9,7 +9,7 @@ tmp_dir = ".air"
 # Just plain old shell command. You could use `make` as well.
 cmd = "go build -o .air/clutch ."
 # Binary file yields from `cmd`.
-bin = ".air/clutch"
+bin = ".air/clutch -env .env.local"
```

Comment:
Since the user will need to create the `.env.local` manually, wont this fail as the file will not exist?

Reply:
Nope, we intentionally ignore errors when the .env files don't exist as it shouldn't block users since these should be optional

Reply:
ok cool i though `filepath.Abs` would check for the file existence but I suppose it does not.